publication-graph-maintenance/producer.js

```diff
@@ -541,16 +541,27 @@ function publicationGraphFilter() {
   return `FILTER ( regex(str(?graph), ${sparqlEscapeString(PUBLICATION_GRAPH)}) )`;
 }
 
-function isConfiguredForExport(triple, config) {
-  const predicate = triple.predicate.value;
-  const rdfType = 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type';
+function isTripleInScopeOfConfiguration(triple, config) {
+
+  const isPredicateRDFType = () => triple.predicate.value === 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type';
+
+  const isGraphOfInterest = () => {
+    if (!config.graphsFilter || config.graphsFilter.length === 0)
+      return true;
+    for (let i = 0; i < config.graphsFilter.length; i++) {
+      if (triple.graph.value.match(config.graphsFilter[i]))
```

Comment:
Sorry, this isn't documented anywhere, but the graphs may be regexes too. See: https://github.com/lblod/app-digitaal-loket/blob/master/config/delta-producer/mandatarissen/export.json#L42

I do realize, this might make these things a tad more complicated.